Test/Case/Model/ShimModelTest.php

```diff
@@ -291,6 +296,9 @@ public function testFindRecursive() {
 	 * Testing missing contain warnings
 	 *
 	 * @expectedException PHPUnit_Framework_Error_Warning
+	 * @expectedExceptionMessage No recursive -1 or contain used for the query in User: Array
+(
```

Comment:
Looks not good.

Reply:
That's what the message is. Placing the `()` to the line above makes the test to fail when run with PHPUnit 5.7.

Reply:
Why? does it need always exp message?

Reply:
I dont think so, if you look into their docs, expectedExceptionMessage is always optional.

Reply:
![screenshot from 2018-04-18 16 31 09](https://user-images.githubusercontent.com/3830106/38938492-ed94de7e-4325-11e8-913a-716569eb529c.png)

Reply:
I didn't find the way to make it optional https://phpunit.de/manual/5.7/en/appendixes.annotations.html#appendixes.annotations.expectedExceptionMessage.

Reply:
:man_facepalming: `@expectedExceptionMessage No recursive -1 or contain used for the query in User` should be actually fine.